src/ui/components/Timeline/Marker.tsx

```diff
@@ -102,4 +103,4 @@ const connector = connect(null, {
 });
 type PropsFromRedux = ConnectedProps<typeof connector>;
 
-export default connector(Marker);
+export default connector( React.memo( Marker, shouldMarkerUpdate ) );
```

Comment:
I bet you can use the `useSelector` hook.

Reply:
Sure, I guess we are not accessing any data from store in this component so usage of `useDispatch` will be appropriate to avoid `connect()` for dispatching actions.